src/base/projects/View.svelte

```diff
@@ -40,18 +39,19 @@
     const seedInstance = profile?.seed ?? result?.seed;
     const cfg = seedInstance && seedInstance.valid ? config.withSeed(seedInstance) : config;
     const info = await proj.getInfo(id, cfg);
-    const urn = isRadicleId(id) ? id : info.meta.urn;
-    const anchors = await getAllAnchors(config, urn, profile?.anchorsAccount ?? addressOrName);
-    let branches = Array([info.meta.defaultBranch, info.head]) as [string, string][];
+    const urn = isRadicleId(id) ? id : info.urn;
+    const anchors = await profile?.confirmedProjectAnchors(urn, config);
```

Comment:
Anchors can eventually be `undefined` here and in the `Source` object we only accept anchors as `string[]`
Maybe we can convert `confirmedProjectAnchors` into a static method?

Reply:
ah I see, not sure how converting it into a static method would help... but we can just do `profile ? profile... : []`

Reply:
Yeah we can do that too.